documentation/user_kit.md

```diff
@@ -261,6 +262,75 @@ Partners can revoke user access to given doors. This will be done by using a par
 
         ⇒ Contact Latch Support
 
+### Get user
+
+Partners can fetch a single user. This will be done by using a partner-scoped token from the BE.
+
+1. GET user request from the Partner BE to the Latch BE
+
+    ```
+    GET https://rest.latchaccess.com/access/sdk/v1/users/:user
+    ```
+
+    HTTP Headers
+	
+    ```
+    Authorization: Bearer {{access_token}}
+    ```
+
+    HTTP Request Body
+	
+    ```
+    <empty>
+    ```
+
+    HTTP Response Body
+
+    ```
+    {
+       "email": "<string>",
+       "firstName": "<string>",
+       "lastName": "<string>",
+       "userUuid": "<string>",,
+       "accesses": [
+         {
+           "doorUuid: "<string>",
+           "passcodeType": "<string>",
+           "shareable": <boolean>,
+           "startTime": "<string>",
+           "endTime": "<string>",
+           "granter": {
+             "type": "<string>",
+             "uuid": "<string>",
+           }
+         },
+         ...
+       ]
+    }
+    ```
+	
+3. If the request was successful, the Partner BE will receive an HTTP 200 containing a User object, with the following fields:
```

Comment:
nit: we can just use enumerations starting at `1.`, to avoid this confusion of having misnumbered items.